github/manager.go

```diff
@@ -107,11 +107,25 @@ func hydrateCommits(q *githubQuery) []Commit {
 			})
 		}
 
+		statusSuccess := bool(edge.Node.StatusCheckRollup.State == githubql.String(githubql.StatusStateSuccess))
```

Comment:
just for readability, what do you think about including this on an _else_ branch?